scripts/auto-stop-idle/autostop.py

```diff
@@ -105,12 +105,7 @@ def get_notebook_name():
             print('Notebook is not idle:', notebook['kernel']['execution_state'])
             idle = False
 else:
-    client = boto3.client('sagemaker')
-    uptime = client.describe_notebook_instance(
-        NotebookInstanceName=get_notebook_name()
-    )['LastModifiedTime']
-    if not is_idle(uptime.strftime("%Y-%m-%dT%H:%M:%S.%fz")):
-        idle = False
+    idle = False # removing check on notebook metadata update time as this could lead to unwanted shutdowns
```

Comment:
by default the script will not shut down the notebook like the autostop script. Open to your suggestion if this script should remove #108-113 as well

Reply:
Could you help me understand why this change was made? I'm a little confused because I don't find the "notebook-instance-monitor" LCC to be depending on this script. So I don't understand why changes are made here, instead of the notebookapi.py script (line 95-101).

Reply:
correct, we were in conversation about this and the PM suggested we add this to autostop. You can ignore this autostop change and focus only on the notebookapi.py script

Reply:
Ok. In that case should we exclude this file from the PR?